### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by surbhi84**
netlify link: https://hallodecor.netlify.app/

### CODE REVIEW

File: carousal.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+let r1 = document.getElementById("radio1");
```
Comment (akshay-rajput) [inline carousal.js]: use proper name here.

File: css/header.css [hunk 2]
```diff
@@ -0,0 +1,117 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  max-width: 100vw;
```
Comment (akshay-rajput) [inline css/header.css]: try using width.. check priority of max-width, width and min-width from documentation.

File: css/header.css [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  font-family: "Libre Baskerville", serif;
+}
+
+.nav-head {
+  position: absolute;
+  min-width: 100%;
+  top: 0;
+  background-color: var(--primary-color-bg);
+  padding: var(--spacing-xs);
+  font-family: "Nunito", sans-serif;
```
Comment (akshay-rajput) [inline css/header.css]: try to keep fonts liimited to 2 or 3 at most. font load takes time and bandwidth, check lighthouse

File: css/homepage.css [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+/* div that contains landscape image with overcard */
+.landscape-overcard {
+  height: 30%;
+  width: 60%;
+  margin-left: 5%;
+}
+
+.overcard {
```
Comment (akshay-rajput) [inline css/homepage.css]: should be called overlay

File: css/homepage.css [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+/* div with overlay text */
+.text-overlay-card {
+  font-size: var(--fs-m);
+  position: absolute;
+}
+
+/* inner div for design implementation */
+.inner-category-overlay:hover {
+  transform: scale(1.05);
```
Comment (akshay-rajput) [inline css/homepage.css]: you can group repeating properties together example .card, .note {transform: scale(0.9)};

File: index.html [hunk 6]
```diff
@@ -4,7 +4,318 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Surbhi Kukreti" />
+    <meta name="description" content="HalloCart Home page" />
+    <link rel="stylesheet" href="/css/common.css" />
+
+    <link rel="stylesheet" href="http://127.0.0.1:5501/halloUi-share.css" />
```
Comment (akshay-rajput) [inline index.html]: remember to change this

File: index.html [hunk 7]
```diff
@@ -4,7 +4,318 @@
...
+            <label for="radio4" class="manual-btn"></label>
+          </div>
+        </div>
+        <!-- end of slides -->
+      </section>
+      <!-- end of carousel -->
+
+      <!-- start of nav-head -->
+      <div class="nav-head">
+        <h2 class="logo">Hallo!Decor</h2>
```
Comment (akshay-rajput) [inline index.html]: turn it into anchor tag and link to homepage.

File: index.html [hunk 8]
```diff
@@ -4,7 +4,318 @@
...
+            <label for="radio3" class="manual-btn"></label>
+            <label for="radio4" class="manual-btn"></label>
+          </div>
+        </div>
+        <!-- end of slides -->
+      </section>
+      <!-- end of carousel -->
+
+      <!-- start of nav-head -->
+      <div class="nav-head">
```
Comment (akshay-rajput) [inline index.html]: use nav tag here

File: index.html [hunk 9]
```diff
@@ -4,7 +4,318 @@
...
+        <h2 class="logo">Hallo!Decor</h2>
+        <!-- start of nav-links -->
+        <ul class="nav-links">
+          <li class="search search-box">
+            <input type="text" placeholder="Search.." class="search-bar" />
```
Comment (akshay-rajput) [inline index.html]: use input type search

File: index.html [hunk 10]
```diff
@@ -4,7 +4,318 @@
...
+            />
+            <div class="text-overlay-card">Bestsellers</div>
+          </div>
+        </div>
+        <!-- big card 2 -->
+        <div class="category-overlay-card">
+          <div class="inner-category-overlay flex-center">
+            <img
+              src="/media/images/basket1.jpg"
+              alt=""
```
Comment (akshay-rajput) [inline index.html]: put some alt text

**Review by akshay-rajput — CHANGES_REQUESTED** [review]
make these changes

File: css/header.css [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+  box-sizing: border-box;
+}
+
+.logo {
+  text-decoration: none;
+  color: black;
+  font-size: var(--fs-m);
+  font-family: "Dancing Script", cursive;
+  border: 4px var(--primary-color);
+  border-style: double;
```
Comment (Naveenchandar) [inline css/header.css]: Border-style can be added in border property.

File: css/header.css [hunk 12]
```diff
@@ -0,0 +1,119 @@
...
+
+.nav-links {
+  gap: var(--spacing-xxl);
+}
+
+.nav-link {
+  padding-right: 0.2rem;
+  border-radius: 0.5rem;
+  border-bottom: 1px solid transparent;
+  border-right: 1px solid transparent;
```
Comment (Naveenchandar) [inline css/header.css]: Try to use shorthand property.

File: css/homepage.css [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+/* different categories of items to browse by */
+.category-card {
+  width: 15%;
+  overflow: hidden;
+  font-family: "Gideon Roman", cursive;
+}
+
+.inner-category-card {
+  cursor: pointer;
+  color: black;
```
Comment (Naveenchandar) [inline css/homepage.css]: You can define a variable for color (with hex code) and utilise it.

File: index.html [hunk 14]
```diff
@@ -4,7 +4,356 @@
...
+    <!-- <link rel="stylesheet" href="http://127.0.0.1:5501/halloUi-share.css" /> -->
+    <link
+      rel="stylesheet"
+      href="https://halloui.netlify.app/halloUi-share.css"
+    />
+
+    <script src="/carousal.js" defer></script>
+    <script
+      src="https://kit.fontawesome.com/c780d21494.js"
+      crossorigin="anonymous"
```
Comment (Naveenchandar) [inline index.html]: font awesome can be used with link tag. Is there any reason of using scripts ?

File: index.html [hunk 15]
```diff
@@ -4,7 +4,356 @@
...
+    </header>
+
+    <!-- start of main content -->
+    <main class="flex-col">
+      <!-- block-1 -->
+      <div class="display-block">
+        <div class="flex-row landscape-overcard">
+          <img
+            src="/media/images/sofaBeige.jpg"
+            alt=""
```
Comment (Naveenchandar) [inline index.html]: Please provide "alt" description for accessibility

File: index.html (hunk 14 above)
Comment (surbhi84) [reply index.html]: i made a kit and it gave me this script tag so i used it

Comment (Naveenchandar) [reply index.html]: Please go through the below link and find exact version of cdn you've used in your project and use it and please verify once the all the font awesome related changes stays the same or different.
https://cdnjs.com/libraries/font-awesome

**Review by Naveenchandar — APPROVED** [review]
Looks good

**Pull request merged**